collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -520,23 +505,23 @@ public static Bitmap getBitmap(String path, BitmapFactory.Options originalOption
         return bitmap;
     }
 
-    public static String readFile(File file) {
-        StringBuilder sb = new StringBuilder();
-        try (BufferedReader br = new BufferedReader(new FileReader(file))) {
-            String line;
-            while ((line = br.readLine()) != null) {
-                sb.append(line).append('\n');
-            }
+    public static byte[] read(File file) {
+        byte[] getBytes = {};
+        try {
+            getBytes = new byte[(int) file.length()];
```

Comment:
This identifier should be a noun, don’t you think? Maybe `bytes`, or `contents`?